client/lib/media/test/actions.js

```diff
@@ -75,14 +75,15 @@ describe( 'MediaActions', () => {
 		MediaActions._fetching = {};
 		window.FileList = function() {};
 		window.FileList.prototype[ Symbol.iterator ] = Array.prototype[ Symbol.iterator ];
-		window.URL = { createObjectURL: sandbox.stub() };
+		savedCreateObjectURL = window.URL.createObjectURL;
+		window.URL.createObjectURL = sandbox.stub();
```

Comment:
So this was here before, but why is it here and why do we have to stub it? I don't see anything obvious in the `media` tests explaining why. Is this part of the new URL parsing? part of the old? what happens if we don't stub it out? is it missing from `jsdom`?

Reply:
The `createTransientMedia` function in `client/lib/media/utils.js` uses `URL.createObjectURL`, which isn't available in Node's `URL`. This is likely because it's part of the [File API](https://w3c.github.io/FileAPI/) and not the [URL standard](https://url.spec.whatwg.org/), even though it gets added as a static method to `URL`.

The reason why the stub was changed in this PR is because it was an incomplete implementation of `URL`. It became a simple object with a `createObjectUrl` method, and nothing else. This way, the full `URL` implementation is preserved (which is needed, since the new code makes full use of `URL`), and we still keep the stub for the `createObjectURL` method.